Apps/Examples/Examples/All Examples/CameraAnimatorsExample.swift

```diff
@@ -51,13 +56,17 @@ public class CameraAnimatorsExample: UIViewController, ExampleProtocol {
         mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
         view.addSubview(mapView)
 
-        // Center the map over New York City.
-        let newYork = CLLocationCoordinate2D(latitude: 40.7128, longitude: -74.0060)
-        mapView.camera.setCamera(to: CameraOptions(center: newYork))
+        mapView.on(.styleLoaded) { [weak self] _ in
+            guard let self = self else { return }
+            // Center the map over New York City.
+            self.mapView.camera.setCamera(to: CameraOptions(center: self.newYork))
+        }
```

Comment:
Why does this need to wait until `.styleLoaded`?

Reply:
As it turns out, the `__map` only starts honoring camera updates once the style is loaded.. I think the reason it used to work earlier is because we were calling `__map.setCamera()` many many more times than we needed to.

Reply:
@alexshalamov any chance GL Native could change this behavior so that the camera can be set at any time?

Reply:
@macdrevx I think we allow camera changes before style is loaded. Modified camera should override style’s default camera.

Reply:
Hmm, I'm definitely not seeing that in my testing.. Will investigate further / open a ticket.

Reply:
This was a red herring. The camera "can" be set at any time.